src/Private/uuidCache.js

```diff
@@ -2,43 +2,30 @@
 const requireFetch = !globalThis.fetch;
 const externalFetch = (...args) => import('node-fetch').then(({ default: fetch }) => fetch(...args));
 const fetch = requireFetch ? externalFetch : globalThis.fetch;
-const cachedUuids = new Map();
+const NodeCache = require('node-cache');
+const cache = new NodeCache();
```

Comment:
I'm just saying but technically map works...

Reply:
Yes a map works but i couldnt work out how to get it to work. I have used node cache before its always been fine with me